src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/imports/importorder/InputImportOrder.java

```diff
@@ -1,22 +0,0 @@
-//non-compiled with javac: contains specially crafted set of imports for testing
```

Comment:
why you removed it ?

Reply:
I just renamed this file to `InputImportOrder1.java` and there are 4 more which are using this file so I created four more with the name `InputImportOrder2.java`, `InputImportOrder3.java`, `InputImportOrder4.java`and `InputImportOrder5.java`